digital-deck-server/services/cardDistService.js

```diff
@@ -53,5 +55,7 @@ function getCards(decks) {
       cards.push(i);
     }
   }
-  return cards;
+ return cardShuffleService.ShuffleCards(
```

Comment:
There's only 1 parameter so you can just do `cardShuffleService.shuffleCards(cards)` in 1 line.